### DISCUSSION TITLE
Using the new Google Nightscout on Loop

### DISCUSSION THREAD

**Discussion by keithgordon4646** [NightscoutFoundation/xDrip#2424] [General]
Hi everyone, I am trying to convert over from Heroku to the new Google server.  We have been successfully using loop as the uploader for nightscout and there is a field that we have to enter a customized URL into Loop.  I entered that about 2-3 years ago with the Heroku site (to be honest I can't even remember what that custom URL was and the app cuts it off, so I can't even see it in it's entirety now).  I am hoping someone can provide me with the URL format to enter my new Google-server-nightscout version so we can make the switch.  Any help is appreciated, and please bear with me as programming and the proper terms are foreign to me.

**Comment by Navid200** [comment]
>  I entered that about 2-3 years ago with the Heroku site (to be honest I can't even remember what that custom URL was and the app cuts it off,

This is the first time I see that you need a variable and you don't know what the correct value for the variable is.
This means that even if you wanted to create a new account on Heroku, you would still have the exact same problem.

There are many alternatives to Heroku for setting up Nightscout:  https://github.com/NightscoutFoundation/xDrip/discussions/2384
Nightscout on Google Cloud is one of them.
Regardless of which one you choose, you are going to have this same exact problem.

Please reach out to those who support your loop setup and find out what variable you need for Nightscout. 

Alternatively, you can pay Heroku $7 a month, and you can keep using it.

**Comment by tzachi-dar** [comment]
@keithgordon4646 can you look at your heroku site and copy the variable from there?

you can also try and getting all variables from the google command line by running:
```
sudo /srv/nightscout-vps/GetHerokuVars.sh
cat heroku_vars.txt
```
You will need to enter your heroku account details.
